detekt/wrapper/src/main/kotlin/io/buildfoundation/bazel/rulesdetekt/wrapper/SandboxExecutor.kt

```diff
@@ -0,0 +1,59 @@
+package io.buildfoundation.bazel.rulesdetekt.wrapper
+
+import java.io.File
+import java.io.FileOutputStream
+import java.io.PrintStream
+import java.nio.charset.Charset
+import java.util.concurrent.atomic.AtomicReference
+
+interface SandboxExecutor {
+
+    data class Result(val code: Int, val stdout: String, val stderr: String)
```

Comment:
Is there an option to stream outputs in the worker protocol? To avoid the memory allocations. If so you can prob pass files here

Reply:
Nope, just `String`.